src/components/Card.js

```diff
@@ -3,18 +3,23 @@ import React from "react"
 import b from "../images/back.svg"
 
 function Card(props) {
-  const {card, front = true, back, Component, height} = props
-  const style = {height}
+  const {card, front = true, back, Component, height, style, className} = props
+  if (style) {
+    if (height !== undefined)
+      style.height = height;
+  }
+  else
+    style = {height};
 
   if (back) {
     return (
-      <img src={b} className="card card__back" alt="card-back" style={style} />
+      <img src={b} className={"card card__back " + (className || "")} alt="card-back" style={style} />
```

Comment:
Here, it's better to use template literals:
```js
className={`card card__back ${className}`}
```
Note, that since above we assigned a default value to `className`, `||` is not needed anymore